README.md

```diff
@@ -1,93 +1,100 @@
-linking information can be found at http://spark-packages.org/package/saurfang/spark-sas7bdat
+## Requirements:
+* [Spark 1.4+ or 2.0+](https://spark.apache.org/downloads.html)
+* [Parso 2.0.10](https://mvnrepository.com/artifact/com.epam/parso/2.0.10)
 
-## Features
+## Download:
 
-This package allows reading SAS files in local or distributed filesystem as
-[Spark DataFrames](https://spark.apache.org/docs/latest/sql-programming-guide.html).
+The latest jar can be downloaded from [spark-packages](http://spark-packages.org/package/saurfang/spark-sas7bdat).
 
-Schema is automatically inferred from meta information embedded in the SAS file.
+## Features:
 
-Thanks to the splittable `SasInputFormat`, we are able to convert a 200GB (1.5Bn rows) .sas7bdat file
-to .csv files using 2000 executors in under 2 minutes.
+* This package allows reading SAS files from local and distributed filesystems, into Spark DataFrames.
+* Schema is automatically inferred from metadata embedded in the SAS file. _(Behaviour can be customised, see parameters below)_
+* The SAS format is splittable when not file-system compressed, thus we are able to convert a 200GB (1.5Bn rows) .sas7bdat file to .csv files using 2000 executors in under 2 minutes.
+* This library uses [parso](https://github.com/epam/parso/) for parsing as it is the only public available parser
+that handles both forms of SAS compression (CHAR and BINARY). 
 
-### SQL API
-SAS data can be queried in pure SQL by registering the data as a (temporary) table.
-
-```sql
-CREATE TEMPORARY TABLE cars
-USING com.github.saurfang.sas.spark
-OPTIONS (path "cars.sas7bdat")
-```
+__NOTE:__ we currently only support reading
+           
+## Docs:
+
+### Parameters:
+* `extractLabel` _(Default: false)_
+  * _Boolean:_ extract column labels as column comments for Parquet/Hive
+* `forceLowercaseNames` _(Default: false)_
+  * Boolean: force column names to lower case
+* `inferDecimal` _(Default: false)_
+  * Boolean: infer numeric columns (with format width >0 and format precision >0), as _Decimal(Width, Precision)_
+* `inferDecimalScale` _(Default: each column's format width)_
+  * Int: scale of inferred decimals
+* `inferFloat` _(Default: false)_
+  * Boolean: infer numeric columns (with <=4 bytes), as _Float_
+* `inferInt` _(Default: false)_
+  * Boolean: infer numeric columns (with <=4 bytes, format width >0 and format precision =0), as _Int_
+* `inferLong` _(Default: false)_
+  * Boolean: infer numeric columns (with <=8 bytes, format width >0 and format precision =0), as _Long_
+* `inferShort` _(Default: false)_
+  * Boolean: infer numeric columns (with <=2 bytes, format width >0 and format precision =0), as _Short_
+* `metadataTimeout` _(Default: 60)_
+  * Int: number of seconds to allow reading of file metadata _(stops corrupt files hanging)_
+* `minPartitions` _(Default: 0)_
+  * Int: minimum number splits to use when reading
+  
```

Comment:
Would it be also possible to add comment you left earlier at https://github.com/saurfang/spark-sas7bdat/pull/44#issuecomment-440840807 - this wasn't obvious for me. thank you.

> the reason for defaulting to Double, is because SAS doesn't have a concept of Longs, instead it just rounds the numeric types on disk when the user looks at it (column formatters). To infer if a column is a long, I check the column formatter, and if its precision is 0, meaning I might reduce precision if someone hasn't defined their formatters correctly.
> 
> (Also, making this change would be inconsistent with previous versions of this reader)
> 
>

Reply:
Done